runtimes/peregrine/src/lib.rs

```diff
@@ -698,7 +698,7 @@ impl parachain_staking::Config for Runtime {
 	type NetworkRewardBeneficiary = runtime_common::SendDustAndFeesToTreasury<Runtime>;
 	type WeightInfo = weights::parachain_staking::WeightInfo<Runtime>;
 
-	const BLOCKS_PER_YEAR: Self::BlockNumber = constants::BLOCKS_PER_YEAR;
+	const BLOCKS_PER_YEAR: BlockNumberFor<Self> = constants::BLOCKS_PER_YEAR;
```

Comment:
Why is this different than what's used for Spiritnet?

Reply:
Good catch: [c4120f6](https://github.com/KILTprotocol/kilt-node/pull/559/commits/c4120f6f440169d5f3b063613ce84b1240a47f31)